screens/request_profile.js

```diff
@@ -0,0 +1,207 @@
+import React from 'react';
+import { StyleSheet, Text, View, Dimensions, Image, FlatList} from 'react-native';
+
+const window = Dimensions.get("window")
+
+let colors = ['#6DC4E0', '#605DF1']
+
+let items = [
+  {
+    name: 'Item',
+    q: 'Q',
+    comment: 'Comments',
+    check: require('./../assets/images/check.png'),
+  },
+  {
+    name: 'Apple',
+    q: 3,
+    comment: 'Granny Smith',
+    check: require('./../assets/images/checkbox.png'),
+  },
+  {
+    name: 'Apple',
+    q: 3,
+    comment: 'Granny Smith',
+    check: require('./../assets/images/checkbox.png'),
+  },
+  {
+    name: 'Apple',
+    q: 3,
+    comment: 'Granny Smith',
+    check: require('./../assets/images/checkbox.png'),
+  }
+]
+
+export default class Request extends React.Component { 
+  render() {
+    return (
+    <View>
+          <View style={styles.container}>
+            <View style={styles.topBanner} />
+            <Image style={styles.shoppingBagicon}
+                source={require('./../assets/images/shopping_bag.png')} />
+            <Image style={styles.personIcon}
+                source={require('./../assets/images/person.png')} />
+            <Image style={styles.arrowIcon}
+                source={require('./../assets/images/back_arrow.png')} />
+            <Text style={styles.browseText}>Browse Other Options</Text>
+            <View style={{justifyContent: 'center', alignItems: 'center'}}>
+              <View style={styles.whiteCard} />
+              <View style={styles.nameBox} />
+              <Text style={styles.nameText}>Bob - CVS</Text>
+           </View>
+           </View>
+                <View style={{top: 315, justifyContent: 'center', alignItems: 'center'}}>
+                  <FlatList
+                  data={items}
+                  keyExtractor={(item, index) => index.toString()}
+                  renderItem={
+                    ({item, index}) => {
+                    return (
+                        <View>
+                        <View style={{opacity: 0.8, height: 50, width: 0.8*window.width, backgroundColor: colors[index % colors.length]}}>
+                        <View style={{padding: 12, flexDirection: 'row', justifyContent: 'space-between'}}>
+                        <Text style={styles.itemText}>{item.name}</Text>
+                        <Text style={styles.itemText}>{item.q}</Text>
+                        <Text style={styles.itemText}>{item.comment}</Text>
+                        <Image style={{width: 22, height: 22}}
+                              source={item.check} />
+                        </View>
+                        </View>
+                        </View>
+                  )
+                  }
+                }
+                />
+              </View>
+              <View style={{justifyContent: 'center', alignItems: 'center'}}>
+              <Text style={styles.dropText}>Approximate Total: $25.00 Drop-Off Location: Baker</Text>
+              <View style={styles.acceptBox} />
```

Comment:
This is not good practice - When we see text within another component (in this case a button) the `<Text>` component should be inside the `<View>` compontent